rocky/onboarding/views.py

```diff
@@ -455,10 +455,14 @@ class OnboardingOrganizationSetupView(
     current_step = 2
     permission_required = "tools.add_organization"
 
+    def get_queryset(self) -> Organization | None:
```

Comment:
This doesn't return a queryset and the `Organization | None` syntax doesn't work on Python 3.8 and 3.9. But given that this code is only used in `get`, I would jus put the code in `get`. It will make the code simpler and easier to understand.